WordPressKit/SiteManagementServiceRemote.swift

```diff
@@ -100,6 +100,27 @@ open class SiteManagementServiceRemote: ServiceRemoteWordPressComREST {
                 failure?(error)
         })
     }
+    
+    /// Trigger a masterbar notification celebrating completion of mobile quick start.
+    ///
+    /// - Parameters:
+    ///   - siteID: The WordPress.com ID of the site.
+    ///   - success: Optional success block
+    ///   - failure: Optional failure block with NSError
+    ///
+    @objc open func markQuickStartAsCompleted(_ siteID: NSNumber, success: (() -> Void)?, failure: ((NSError) -> Void)?) {
```

Comment:
imo we should clarify _what_ part of Quick Start is being completed. Right now there are ways to complete each part of the quick start checklist, and this is completing the whole checklist itself. I'd prefer something like `markQuickStartChecklistAsComplete`, but if that's just to long then so be it.

Reply:
Totally agree with you @nheagy !